### ISSUE TITLE
`jaws deploy resources` is missing

### ISSUE THREAD

**Issue by boushley** [jaws-framework/JAWS#154]
We need to add the `deploy resources` command again.

**Comment by ac360** [comment]
Thanks @boushley.  I'm starting this right now!

**Comment by ac360** [comment]
Fixed in a2dce9c8be94f39f6f71d2fbaa22d0c512ff056a  Going to update to JAWS v1.2 today.

**Comment by boushley** [comment]
Great thanks!
